### PR TITLE
Set ia to ffff

### PR DESCRIPTION
**Pull request by FrankZRS**

### CODE REVIEW

File: api/oc_knx_dev.c [hunk 1]
```diff
@@ -1440,7 +1441,7 @@ oc_knx_device_storage_reset(size_t device_index, int reset_mode)
     // load state: unloaded, and programming mode is true
     oc_knx_lsm_set_state(device_index, LSM_S_UNLOADED);
     // set the other data to null
```
Comment (WAvdBeek) [inline api/oc_knx_dev.c:1443]: please put here "KNX defaults"

File: api/oc_knx_dev.c [hunk 2]
```diff
@@ -1459,11 +1460,11 @@ oc_knx_device_storage_reset(size_t device_index, int reset_mode)
       : Fixed : 00h */
     oc_storage_erase(KNX_STORAGE_IA);
     // set the ia to zero
```
Comment (WAvdBeek) [inline api/oc_knx_dev.c:1462]: here as well

**Review by WAvdBeek — COMMENTED** [review]
fix the comment, use "KNX defaults" instead of setting to zero..

**Comment by FrankZRS** [comment]
~format

**Pull request merged**